src/main/java/com/revature/models/User.java

```diff
@@ -1,19 +1,35 @@
 package com.revature.models;
 
-import lombok.*;
+import java.io.Serializable;
+
+import javax.persistence.CascadeType;
```

Comment:
This is a very opinionated concern, but I think it might be better to leave the imports as they were originally, using javax.persistence.* and javax.validation.* to reduce the number of import statements.